daemon/container_unix.go

```diff
@@ -640,7 +640,7 @@ func (container *Container) updateNetworkSettings(n libnetwork.Network, ep libne
 		return err
 	}
 
-	if container.hostConfig.NetworkMode == runconfig.NetworkMode("bridge") {
+	if container.hostConfig.NetworkMode == runconfig.NetworkMode("default") {
 		networkSettings.Bridge = container.daemon.config.Bridge.Iface
```

Comment:
On Sat, Aug 22, 2015 at 11:46 AM, Madhu Venugopal <notifications@github.com>
wrote:

> In daemon/container_unix.go
> <https://github.com/docker/docker/pull/15747#discussion_r37696809>:
>
> > @@ -640,7 +640,7 @@ func (container *Container) updateNetworkSettings(n libnetwork.Network, ep libne
> >  		return err
> >  	}
> >
> > -	if container.hostConfig.NetworkMode == runconfig.NetworkMode("bridge") {
> > +	if container.hostConfig.NetworkMode == runconfig.NetworkMode("default") {
> >  		networkSettings.Bridge = container.daemon.config.Bridge.Iface
>
> @mrjana <https://github.com/mrjana> +1
>

Do you think I need to split the patch and submit this one as a fix to the
bridge output of "docker inspect"?

- Hui


> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/15747/files#r37696809>.
>